### PR TITLE
[CFE-528]: Feature/asset selector

### PR DESCRIPTION
**Pull request by Poafs1**

### CODE REVIEW

File: src/lib/components/token/TokenImageRender.tsx [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
     boxSize={boxSize}
     fallback={<NAToken boxSize={boxSize} />}
     fallbackStrategy="beforeLoadOrError"
     {...props}
   />
 );
 
-interface TokenImageRenderProps extends ImageProps {
+interface TokenImageRenderProps extends ChakraImageProps {
   logo: Option<string> | Option<string>[];
```
Comment (evilpeach) [inline src/lib/components/token/TokenImageRender.tsx:20]: `logo: Option<string | string[]>;`

File: src/lib/components/forms/index.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-export * from "./AssetInput";
+export * from "./assetInput";
```
Comment (evilpeach) [inline src/lib/components/forms/index.ts:1]: asset-input

File: src/lib/components/fund/SelectFund.tsx [hunk 3]
```diff
@@ -26,30 +36,116 @@ export const SelectFund = ({
   assetsSelect,
   labelBgColor,
 }: SelectFundProps) => {
+  const { address } = useCurrentChain();
+  const { data: balances } = useBalances(address as BechAddr20);
```
Comment (songwongtp) [inline src/lib/components/fund/SelectFund.tsx:40]: make the input of `useBalances` optional, then handle `undefined` like
```
() => {
    if (!address) throw new Error("address is undefined (useBalances)")

    return isSei
        ? getBalances(endpoint, address)
        : getBalancesLcd(endpoint, address)
}
```

File: src/lib/components/fund/SelectFund.tsx [hunk 4]
```diff
@@ -26,30 +36,116 @@ export const SelectFund = ({
...
   const { data: assetInfos = [] } = useAssetInfoList({ assetType: "native" });
```
Comment (songwongtp) [inline src/lib/components/fund/SelectFund.tsx:41]: remove this

File: src/lib/components/fund/SelectFund.tsx [hunk 5]
```diff
@@ -26,30 +36,116 @@ export const SelectFund = ({
...
+  const { data: assetInfosWithPrices } = useAssetInfos({ withPrices: true });
```
Comment (songwongtp) [inline src/lib/components/fund/SelectFund.tsx:42]: data: assetInfos

**Review by songwongtp — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]

**Pull request merged**